tgui/eslint.config.mjs

```diff
@@ -16,156 +16,173 @@ const compat = new FlatCompat({
     allConfig: js.configs.all
 });
 
-export default [...compat.extends("prettier"), {
-    ignores: ["/.yarn/**", "/**/node_modules", "/**/*.bundle.*", "/**/*.chunk.*", "/**/*.hot-update.*", "/packages/inferno/**"],
+export default [
+  ...compat.extends('prettier'),
+  {
+    ignores: [
+      '**/.yarn/**',
+      '**/node_modules',
+      '**/*.bundle.*',
+      '**/*.chunk.*',
+      '**/*.hot-update.*',
+      '**/packages/inferno/**',
+    ],
```

Comment:
for reviewers, this is the only actual change. Everything below is formatting changes